docker-build/lom-slave-bullseye/Dockerfile

```diff
@@ -0,0 +1,475 @@
+FROM debian:bullseye
+
+MAINTAINER gulv@microsoft.com
+
+COPY ["no-check-valid-until", "/etc/apt/apt.conf.d/"]
+COPY ["apt-multiple-retries", "/etc/apt/apt.conf.d/"]
+COPY ["sources.list.amd64", "/etc/apt/sources.list"]
+COPY ["rsyslog.conf", "/etc/"]
+COPY ["lom_start.sh", "/usr/local/bin/"]
+
```

Comment:
Will we be copying the lom source code in our custom dockerfile ?

Reply:
We mount the dir. No copying